changelog/6354.feature.md

```diff
@@ -0,0 +1,4 @@
+The Rasa Open Source API endpoint `POST /model/train` now supports training data in YAML
+format. Please specify the header `Content-Type: application/yaml` when
+training a model using YAML training data. 
+See the [API documentation](./api/http-api.mdx) for more information.
```

Comment:
Should we say a bit more about how the new request parameters affect the `force` and `default_directory` keys in the json payload?

Reply:
added another changelog

Reply:
don't think that's checked in yet

Reply:
🙈